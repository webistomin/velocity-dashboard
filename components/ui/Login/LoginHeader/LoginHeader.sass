+b('login')
  +e('header')
    padding: rem(26px) 0

    +media($screen-sm)
      padding: rem(58px)

    +media($screen-lg)
      padding: rem(38px)

  +e('header-container')
    display: flex
    justify-content: center

    +media($screen-sm)
      padding-left: 0
      padding-right: 0

    +media($screen-lg)
      width: rem(320px)
      justify-content: flex-start
