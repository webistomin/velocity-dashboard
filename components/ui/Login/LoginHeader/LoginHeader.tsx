import { VueComponent } from 'types/vue-components';
import { Component } from 'nuxt-property-decorator';
import { VNode } from 'vue';

import BaseLogo from 'components/base/BaseLogo';

import './LoginHeader.sass';

@Component({
  name: 'LoginHeader',
})
export default class LoginHeader extends VueComponent {
  render(): VNode {
    return (
      <header class='login__header'>
        <div class='login__header-container container'>
          <BaseLogo />
        </div>
      </header>
    );
  }
}
