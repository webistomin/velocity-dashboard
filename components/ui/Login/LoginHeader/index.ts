import LoginHeader from './LoginHeader';

export default LoginHeader;
