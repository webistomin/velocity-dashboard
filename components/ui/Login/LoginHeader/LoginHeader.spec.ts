import { shallowMount } from '@vue/test-utils';
import LoginHeader from './LoginHeader';

describe('LoginHeader', () => {
  const factory = (propsData = {}, options = {}) => {
    return shallowMount(LoginHeader, {
      propsData: {
        ...propsData,
      },
      ...options,
    });
  };

  it('Matches snapshot', () => {
    const wrapper = factory().html();
    expect(wrapper).toMatchSnapshot();
  });
});
