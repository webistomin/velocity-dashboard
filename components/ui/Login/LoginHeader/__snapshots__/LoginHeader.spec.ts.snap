// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginHeader Matches snapshot 1`] = `
"<header class=\\"login__header\\">
  <div class=\\"login__header-container container\\">
    <vuecomponent-stub></vuecomponent-stub>
  </div>
</header>"
`;
